hpcbench/toolbox/process.py

```diff
@@ -76,7 +76,9 @@ def physical_cpus():
 
 def build_slurm_arguments(argdict):
     args = []
-    for k, v in argdict.items():
+    keys = sorted(argdict.keys())
```

Comment:
why are you doing this? could you not just use OrderedDict if you want to enforce sorted keys?

Reply:
Arguments are sorted only to ease validation in unit-tests.
AFAIK `OrderedDict` keeps insertion order, so I don't know how it can be useful here.